src/build/internal/sass.ts

```diff
@@ -1,20 +1,18 @@
-import node_sass from "node-sass";
 import node_sass_utils from "node-sass-utils";
+import sass from "sass";
 import { isString } from "ts-type-guards";
 
-const SassUtils = node_sass_utils(node_sass);
+const SassUtils = node_sass_utils(sass);
 
-interface IKeyString {
-    getValue: () => unknown;
-}
-
-export function getGlobalFrom(objectToBeExposedToSass: object): (keyString: IKeyString) => any {
+export function getGlobalFrom(objectToBeExposedToSass: object): (keyString: sass.types.SassType) => sass.types.SassType | void {
```

Comment:
Yep that's true. According to the [node-sass-utils](https://github.com/sass-eyeglass/node-sass-utils#sassutilscasttosassjsvalue) documentation `castToSass` and to the [code](https://github.com/sass-eyeglass/node-sass-utils/blob/e5c33655532dd0c4fa65428a1a17a8f1e1b468fa/lib/coercion.js#L9) `castToSass` should always return a valid sassType or throw an exception as well (so we might also want to change the node-sass-utils.d.ts definition).

The `never` type is 'absorbed' by the return type so for: `function(): sass.types.SassType | never` the `never` type is only useful for 'reference' (so the developer knows that the function may throw an exception). I don't know if it's a good practice to explicitly use `never` for this case of 'an exception might occur' though.

What do you think is the best? Should we go for `SassType | never` or simply `SassType`?

Reply:
Yes, we probably want to update `castToSass` in `src/build/node-sass-utils.d.ts`.

I personally like the idea of using `never` to indicate that a function may throw, but AFAIK it's not idiomatic and we wouldn't be able to keep it in sync with the actual behavior (i.e. if a function is changed so that it doesn't throw anymore, it's easy to forget to remove `| never` from its type signature).

Reply:
I have updated `castToSass` and removed `| never`.